_episodes/05-starting-jekyll.md

```diff
@@ -7,75 +7,351 @@ questions:
+> a single place (even if you only use them once). In combination with Jekyll layouts/templates (to be covered in the next episode) they are a great way of creating reusable markup snippets that can be repeated on multiple or even on every page of your website. Reuse helps you reduce the amount of code you have to write.  
+{: .callout}
+
+### List Parameters and For Loops
+
+YAML lets us define lists and dictionaries too, in addition to simple parameters we have seen so far. 
+And Jekyll supports the use of these more complex parameters as well. 
+
+For example, we may want
+to list the names of all the project team members, together with their roles and start dates inside the `_config.yml` file
+and make use of this information throughout the website. To do so in YAML notation would look something like:
+
+~~~
+team_members: [
+    {name: "Sarah Becker", role: "project lead", start_date: "2016-02-01"},
+    {name: "Jane Smith", role: "maintainer", start_date: "2018-03-15"},
+    {name: "Albert Hamilton", role: "editor", start_date: "2017-12-01"},
+]
+~~~
+{: .language-yaml}
+This defines 'team_members' as a list of 3 elements; each element is a dictionary with key-value pairs for name, role and date.
+
+> ## Indentation in YAML
+> Note that indentation level in YAML is important - it can be a cause of some not-so-obvious mistakes. Use online YAML
+>code validators, such as [YAML Lint](http://www.yamllint.com/), to make sure your YAML is correctly formatted.
+>
+{: .callout}
+
+In order to access all elements in a list you need a 'for loop', which is performed in Liquid 
+by surrounding it in a curly brace percentage sign as: 
+`{% raw %}{% for variable in list %}{% endraw %}`. You follow this with the 'body' of your loop, and finish off
+ with the 'for loop' end-tag: `{% raw %}{% endfor %}{% endraw %}`.
+ 
+Let's access the information about the team and display the name of the project lead on the home page `index.md` and the information about the whole team in `about.md`. Note that we access the team lead's name as `team_members[0]["name"]` because the team lead is the first team member in the `team_members` list.
+
+1. Modify `_config.yml` file and add the `team_members` parameter as defined above. The file should now look like:
+
+    ~~~
+    title: "Building Websites in GitHub"
+    description: "This research project develops training materials for reseachers wanting to learn to build project
+    websites in GitHub with GitHub Pages."
+    email: "team@my.research.org"
+    twitter: "https://twitter.com/my_research_project"
+    team_members: [
+        {name: "Sarah Becker", role: "project lead", start_date: "2016-02-01"},
+        {name: "Jane Smith", role: "maintainer", start_date: "2018-03-15"},
+        {name: "Albert Hamilton", role: "editor", start_date: "2017-12-01"}
+        ]
+    ~~~
+    {: .language-yaml}
+
+2. In file `index.md` add the team lead's name so that it looks like:
```

Comment:
Maybe switch 2 and 3 here?  Make the list of team members in the team section first and then link to it in the index.

Reply:
Fixed via 5465275.